src/OrchardCore.Modules/OrchardCore.Forms/ViewModels/TextAreaPartEditViewModel.cs

```diff
@@ -2,6 +2,19 @@ namespace OrchardCore.Forms.ViewModels;
 
 public class TextAreaPartEditViewModel
 {
+    private const int DefaultRows = 10;
```

Comment:
I think this is very large by default. I would figure out a number that is very close to the current experience (probably 3 or 4), and use that value by default to avoid changing the existing behavior. 

Also, we use for all private properties including private constants.

Reply:
Please check the related issue https://github.com/OrchardCMS/OrchardCore/issues/17280#issuecomment-2568185194

Reply:
I agree with @sebastienros comment. But your PR force 10 by default. In the view, conditionally add the attribute and set the default to 3 or 4 but not 10.